tsdata/acs.py

```diff
@@ -18,33 +17,33 @@
 from django.db import transaction
 
 
-# Variables: http://api.census.gov/data/2014/acs5/variables.html
+# Variables: http://api.census.gov/data/2016/acs/acs5/variables.json
 RACE_VARIABLES = {
```

Comment:
Based on client feedback it seems to me that these variables need to be updated for NC only, while the other states remain the same.